### DISCUSSION TITLE
Is it possible to output more than 1 protein model? If so, how?

### DISCUSSION THREAD

**Discussion by 1000000000000000000000000000000** [facebookresearch/esm#452] [General]
Hi all,

Is it possible to have ESM output more than one protein structure prediction? I want to generate an ensemble of possible protein conformations using this tool. Any advice or thoughts?

Thanks all!

**Comment by tomsercu** [comment]
You could provide multiple masked input sequences to the LM, with varying masking patterns. No guarantees of the quality and/or variations of those predictions though.

**Reply by 1000000000000000000000000000000** [reply]
Thanks for the helpful reply @tomsercu !

Apologies for the simple question but when you wrote about 'masking patterns', what exactly did you mean? 

Are these 'in-silico mutations' that one would insert into the wild-type sequence or something different?

Thanks again!
